racer.el

```diff
@@ -123,6 +124,10 @@
       (substring s (+ idx (length needle)))
     s))
 
+(defun racer-complete--prefix-p (beg end)
+  "Return if a completion should be triggered for a prefix between BEG and END."
```

Comment:
I think that's implied since it's a predicate function, but I can add that if it's unclear.